packages/backend/src/index.ts

```diff
@@ -19,6 +20,7 @@ export type ClerkOptions = CreateBackendApiOptions &
 
 export function Clerk(options: ClerkOptions) {
   const opts = { ...options };
+  assertValidSecretKey(opts.secretKey || opts.apiKey);
```

Comment:
⛔ this check should not exist since the `secretKey` could be passed as runtime parameter in authenticateRequest.  If we add this call here, it will fail in `Cleck` instance inititialization.

Reply:
Based on the above comment (if we remove this line) the functionality of the code will not change. Let's add a warning in the `assertValidSecretKey` to have a kind of indication in the logs of our application (nextJS, remix) and see how we can tackle it after a discussion.